syntax = "proto3";

service RemoteBoardIO {
  rpc SendSensorData (SensorRequest) returns (SensorResponse) {}
}

message SensorRequest {
}

message SensorResponse {
  repeated int32 sensorValues = 1;
}